warp/cli/warp_cli.py

```diff
@@ -27,6 +27,12 @@ def warp():
 @click.argument("file_path", type=click.Path(exists=True))
 @click.argument("contract_name")
 def transpile(file_path, contract_name):
+    """
+    FILE_PATH: Path to your solidity contract\n
+    CONTRACT_NAME: Name of the primary contract (non-interface, non-library, non-abstract contract) that you wish to transpile
+
+    Generates the transpiled Cairo contract.
```

Comment:
Well, now it generates a json with all the info needed to use that contract, including the transpiled Cairo contract.